api/swagger.yaml

```diff
@@ -2882,6 +2882,11 @@ definitions:
             type: "object"
             additionalProperties:
               type: "string"
+          # Applicable to UNIX platforms
```

Comment:
Might be better to include this as a line in the description (perhaps change UNIX to Linux, or change to, e.g., "This option is not used by Windows containers")

Reply:
Changed